php/stats.php

```diff
@@ -152,85 +174,89 @@
 (SELECT 'W' dir, a.* FROM visited_airports a ORDER BY x ASC LIMIT 1)
 SQL;
 
-$compass_labels = [
-    'N' => _("Northernmost"),
-    'S' => _("Southernmost"),
-    'E' => _("Easternmost"),
-    'W' => _("Westernmost")
-];
-
 $sth = $dbh->query($sql);
```

Comment:
We should pass `PDO::FETCH_ASSOC` as the second parameter, as we don't need the 0 based indexed columns anymore.